awx/ui_next/src/components/UserAccessAdd/resources.data.jsx

```diff
@@ -0,0 +1,208 @@
+import { t } from '@lingui/macro';
+import {
+  JobTemplatesAPI,
+  WorkflowJobTemplatesAPI,
+  CredentialsAPI,
+  InventoriesAPI,
+  ProjectsAPI,
+  OrganizationsAPI,
+} from '../../api';
+
+export default function getResourceTypes(i18n) {
```

Comment:
> * There is no loading indicator when loading second page of the prompt sequence. In general we _always_ need loading indicators when fetching content but it is particularly noticeable on this view when I tried it because the api requests involved are relatively heavy and take a moment to load.

Yes I can add a loading indicator.  That component that creates that second view needs to be brought up to date and implement hooks.  I was going to do that, actually had the code written out for it, but decided that it didn't really belong in this PR.  Now I have an excused to bring it up to do and fix that loading issue!!

> * Applying search filters to the list of resources doesn't seem to work

That's weird. I wonder whats going on there.  Will dig in.